packages/react-magma-dom/src/components/Table/Table.tsx

```diff
@@ -122,6 +128,12 @@ export const TableContainer = styled.div<{
           : props.theme.colors.focus};
   }
 `;
+export const StyledCaption = styled.caption<{ isInverse: boolean; tableTitleNode: boolean; }>`
+  ${headingMediumStyles};
+  margin: ${props => props.tableTitleNode ? `` : props.theme.spaceScale.spacing04};
```

Comment:
```suggestion
  margin: ${props => props.tableTitleNode || props.theme.spaceScale.spacing04};
```

The logical OR operator || can be used to check if props.tableTitleNode is truthy. If it is truthy, an empty string is used as the value of margin. If props.tableTitleNode is falsy, props.theme.spaceScale.spacing04 is used as the value of margin

OR

you can leave as is but you can just use `""` instead of ticks